client/my-sites/plugins/disconnect-jetpack/disconnect-jetpack-dialog.jsx

```diff
@@ -1,83 +1,95 @@
 /**
  * External dependencies
  */
-var page = require( 'page' ),
-	React = require( 'react' );
+import page from 'page';
+import React, { Component } from 'react';
+import { translate } from 'i18n-calypso';
```

Comment:
Oh right, interesting. 👍  We can preserve the ref from the localize side, but I'm not sure we'll want to do that now. I think we might need to do this eventually for things like popover usages.